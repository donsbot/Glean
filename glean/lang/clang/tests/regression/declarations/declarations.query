query: |
    R where cxx1.DeclarationSrcRange { _, R }
transform: [gensort, []]
