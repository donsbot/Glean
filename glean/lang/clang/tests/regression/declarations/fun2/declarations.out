[
  "@generated",
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 9,
      "columnBegin": 1,
      "lineEnd": 19,
      "columnEnd": 1
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 10,
      "columnBegin": 3,
      "lineEnd": 10,
      "columnEnd": 7
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 3,
      "lineEnd": 11,
      "columnEnd": 17
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 11,
      "columnBegin": 7,
      "lineEnd": 11,
      "columnEnd": 16
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 12,
      "columnBegin": 3,
      "lineEnd": 12,
      "columnEnd": 8
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 14,
      "columnBegin": 3,
      "lineEnd": 14,
      "columnEnd": 17
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 15,
      "columnBegin": 3,
      "lineEnd": 15,
      "columnEnd": 33
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 16,
      "columnBegin": 3,
      "lineEnd": 16,
      "columnEnd": 22
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 16,
      "columnBegin": 18,
      "lineEnd": 16,
      "columnEnd": 21
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 17,
      "columnBegin": 3,
      "lineEnd": 17,
      "columnEnd": 22
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 17,
      "columnBegin": 18,
      "lineEnd": 17,
      "columnEnd": 21
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 18,
      "columnBegin": 3,
      "lineEnd": 18,
      "columnEnd": 19
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 21,
      "columnBegin": 1,
      "lineEnd": 21,
      "columnEnd": 34
    }
  },
  {
    "key": {
      "file": { "key": "test.cpp" },
      "lineBegin": 21,
      "columnBegin": 22,
      "lineEnd": 21,
      "columnEnd": 33
    }
  }
]